data/campaigns/Northern_Rebirth/scenarios/02_01_Infested_Caves.cfg

```diff
@@ -552,7 +552,7 @@
             [else]
                 [message]
                     speaker=unit
-                    message= _ "Hey look, it’s an ancient door. It seems the skeletons were too stupid to find how to open it."
+                    message= _ "Hey look, it’s an ancient door. It seems the skeletons were too stupid to find out how to open it."
```

Comment:
If we have both a "find out" and "figure out" version, then two strings go to the translators. If that's the intention, then please add a po: comment about it.